content/documentation/tutorials/nats-source.md

```diff
@@ -58,12 +65,20 @@ cd gnatsd
 go build
 ```
 
-**6. Run the unit regression tests.**
+**7. Start nats-server.**
+
+```
+./gnatsd
+```
+
+<!-- **6. Run the unit regression tests.**
 
 Optionally you can run the unit regression tests:
 
 ```
 go test ./...
-```
+``` -->
+
+**A successful build run produces no messages and creates an executable called `gnatsd` in this directory.**
 
-A successful build run produces no messages and creates an executable called `gnatsd` in this directory. You can invoke this binary, with no options and no configuration file, to start a server with acceptable standalone defaults (no authentication, no clustering).
+**You can invoke this binary, with no options and no configuration file, to start a server with acceptable standalone defaults (no authentication, no clustering).**
```

Comment:
I'd change this to:

    ... and no configuration file for local testing. To start a server with acceptable production 
    defaults please read our (README.md)[https://github.com/nats-io/gnatsd#command-line-
    arguments].

Make sense?